measure_extinction/stardata.py

```diff
@@ -884,15 +885,11 @@ def read(self):
                 elif cpair[0] in poss_mod_params:
                     self.model_params[cpair[0]] = cpair[1]
                 elif cpair[0] == "corfac_spex_SXD":
-                    if cpair[1] == "None":
-                        self.corfac["SpeX_SXD"] = None
-                    else:
-                        self.corfac["SpeX_SXD"] = float(cpair[1])
+                    self.corfac["SpeX_SXD"] = eval(cpair[1])
```

Comment:
Why `eval` instead of `float`?  Curious and want to learn.  Never used eval before.

Reply:
cpair[1] is always a string, as that is how the code reads from the .dat files. If cpair[1] is a number, float works perfectly fine to convert the string to a float. However, if cpair[1] is None (when the scaling factor could not be calculated because there was no photometry, which was the case before I added the WISE data), then `float` will fail: 
`float("None")` will give an error. Before, I worked around this using an extra if-statement to check `if cpair[1] == None`. However, I "discovered" that `eval` can do both steps. It converts the strings to whatever they present. A number will be converted to a float and "None" will be converted to NoneType, which is exactly what is needed in the rest of the code. I hope this is clear. :)

Reply:
Thats really cool.  Thanks for the informative answer!